<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NeuroInterface Adapter</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>741</width>
      <height>193</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="keysLabel">
       <property name="text">
        <string>Назначенные клавиши</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="keysLine_3">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="keysLine_4">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="keysLine_2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="changeButton_2">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="movementLine_2">
       <property name="text">
        <string>Разжать руку 1</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="keysLine">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="movementLine_3">
       <property name="text">
        <string>Сжать руку 2</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="movementLabel">
       <property name="text">
        <string>Движение</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="changeButton_4">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="movementLine">
       <property name="text">
        <string>Сжать руку 1</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="movementLine_4">
       <property name="text">
        <string>Разжать руку 2</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="changeButton_3">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="changeButton">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Стоп</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Не запущена</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="settings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="exportSets"/>
    <addaction name="importSets"/>
    <addaction name="profile"/>
   </widget>
   <widget class="QMenu" name="server">
    <property name="title">
     <string>Сервер</string>
    </property>
    <addaction name="changeIP_Port"/>
   </widget>
   <widget class="QMenu" name="about">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutProgram"/>
   </widget>
   <addaction name="settings"/>
   <addaction name="server"/>
   <addaction name="about"/>
  </widget>
  <action name="exportSets">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Экспорт настроек</string>
   </property>
  </action>
  <action name="importSets">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Импорт настроек</string>
   </property>
  </action>
  <action name="profile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Выбрать профиль</string>
   </property>
  </action>
  <action name="changeIP_Port">
   <property name="text">
    <string>Изменить IP/Port</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="aboutProgram">
   <property name="text">
    <string>О NeuroApp</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
